inception-doc/src/main/resources/META-INF/asciidoc/developer-guide/setup.adoc

```diff
@@ -225,6 +225,13 @@ way is to go to `File -> Open` and the select the `pom.xml` in the {product-name
 IntelliJ IDEA will then guide you through the import process, the defaults work out of the box.
 INCEpTION can now be started via running `inception-app-webapp/src/main/java/de/tudarmstadt/ukp/inception/INCEpTION.java`.
 
+If you get errors that certain classes are not found, then open a terminal, go to the INCEpTION
+repository root and run
+
+    mvn clean install -DskipTests=true -D "checkstyle.skip=true"
```

Comment:
Shouldn't that be `-Dcheckstyle.skip=true`? Why the quotes and the space after `-D`?

Reply:
I changed it,both works